lib/packages/core/lib/src/views/media/media_preview.dart

```diff
@@ -239,42 +238,20 @@ class _LMChatMediaPreviewScreenState extends State<LMChatMediaPreviewScreen> {
     );
   }
 
-  Container _defVideo() => Container();
+  LMChatVideo _defVideo() => LMChatVideo(
+        media: mediaList[currPosition],
+        key: ObjectKey(mediaList[currPosition].hashCode),
+      );
 
   Widget _defVideoThumbnail(int index) {
-    return mediaList[index].thumbnailFile != null
-        ? ClipRRect(
-            borderRadius: BorderRadius.circular(8.0),
-            child: LMChatImage(
-              imageFile: mediaList[index].thumbnailFile!,
-              style: const LMChatImageStyle(
-                boxFit: BoxFit.cover,
-              ),
-            ),
-          )
-        : ClipRRect(
-            borderRadius: BorderRadius.circular(8.0),
-            child: FutureBuilder(
-              future:
-                  getVideoThumbnail(mediaList[index].toAttachmentViewData()),
-              builder: (context, snapshot) {
-                if (snapshot.connectionState == ConnectionState.waiting) {
-                  return mediaShimmer();
-                } else if (snapshot.data != null) {
-                  return Image.file(
-                    snapshot.data!,
-                    fit: BoxFit.cover,
-                  );
-                } else {
-                  return SizedBox(
-                    child: Icon(
-                      Icons.error,
-                      color: LMChatTheme.theme.secondaryColor,
-                    ),
-                  );
-                }
-              },
-            ),
-          );
+    return ClipRRect(
```

Comment:
Could we Wrap `LMChatImage` to `ClipRRect` inside the `LMChatImage` itself, and can customise their borderRadius by style class, by this we can remove external wrapping every time. and also we will achieve the return type as `LMChatImage` needed for customisation.